docs/3.0-migration-guide.d/modify_MBEDTLS_ECP_FIXED_POINT_OPTIM_behaviour.md

```diff
@@ -0,0 +1,8 @@
+Change MBEDTLS_ECP_FIXED_POINT_OPTIM behaviour
+------------------------------------------------------
+
+The option MBEDTLS_ECP_FIXED_POINT_OPTIM now use more ROM and does not increase
+peak RAM usage anymore.
+
+If you are limited by ROM space, you can define MBEDTLS_ECP_FIXED_POINT_OPTIM
+to `0` in your config file. This will save about 50 KiB ROM space.
```

Comment:
Thanks for providing numbers, I think that's useful to help people make decisions about what options to use. However in that case I'm not sure this is the best way to present things: people who have tight constraints on code size are likely to only have 1 or 2 curves enabled (say P-256 and perhaps P-384), so the savings are not going to be that large.

On the other hand, we don't want to overwhelm readers with overly detailed information. Perhaps something like "The impact depends on the number and size of enabled curves. For example, for P-256 the difference is 1KB; see the documentation of this option for details."

Reply:
fixed